.travis.yml

```diff
@@ -14,4 +14,11 @@ before_install:
   - go get github.com/modocache/gover
 
 script:
+  - go build cmd/activity-monitor/main.go
```

Comment:
These should probably be in test.sh so devs will catch problems before they go to Travis.